contracts/payloads/IGP80/PayloadIGP80.sol

```diff
@@ -368,6 +376,36 @@ contract PayloadIGP80 is PayloadIGPConstants, PayloadIGPHelpers {
         FLUID_RESERVE.approve(protocols, tokens, amounts);
     }
 
+    // @notice Action 10: Update USDe-USDT DEX Lower Range and Fee %
+    function action10() internal {
+        address USDe_USDT_DEX_ADDRESS = getDexAddress(18);
+
+        {
+            // Update Lower Range
+            IFluidDex(USDe_USDT_DEX_ADDRESS).updateRangePercents(
+                0.15 * 1e4, // 0.15%
+                0.15 * 1e4, // 0.15%
+                2 days
+            );
+        }
+    }
+
+    // @notice Action 11: Update sUSDe-USDT DEX Trading Fee
+    function action11() internal {
+     {
+        address sUSDe_USDT_DEX_ADDRESS = getDexAddress(15);
+        
+            if (!PayloadIGP80(ADDRESS_THIS).skipAction11()){
```

Comment:
simplify code by doing same as in action9 

        if (PayloadIGP80(ADDRESS_THIS).skipAction11()) return;

Reply:
cools